WordPress/Tests/WP/EnqueuedResourceParametersUnitTest.1.inc

```diff
@@ -98,3 +98,13 @@ wp_register_script( 'someScript-js', 'https://example.com/someScript.js' , array
 
 // Safeguard handling of non-lowercase `null`.
 wp_register_script( 'someScript-js', 'https://example.com/someScript.js' , array( 'jquery' ), NULL, true ); // Error - 0, false or NULL are not allowed.
+
+/*
+ * Safeguard handling of fully qualified \true, \false and \null.
+ * Also safeguard that adding T_NS_SEPARATOR to $false_tokens doesn't cause false positives due to problems in is_falsy().
+ */
+wp_register_script( 'someScript-js', 'https://example.com/someScript.js' , array( 'jquery' ), \FALSE, true ); // Error - 0, false or NULL are not allowed.
+wp_register_script( 'someScript-js', 'https://example.com/someScript.js' , array( 'jquery' ), \null, true ); // Error - 0, false or NULL are not allowed.
+wp_register_script( 'someScript-js', 'https://example.com/someScript.js' , array( 'jquery' ), \Null, true ); // Error - 0, false or NULL are not allowed.
+wp_register_script( 'someScript-js', 'https://example.com/someScript.js' , array( 'jquery' ), \true, true ); // Ok.
+wp_register_script( 'someScript-js', 'https://example.com/someScript.js' , array( 'jquery' ), \get_version(), true ); // OK.
```

Comment:
For the fourth (`$ver`) parameter, these tests cover the cases sufficiently.

For the fifth parameter, we are not really looking at the value, but as it's typically a boolean anyway, we may as well have some case and FQN variations in these test case lines for the fifth parameter. Just as a safeguard in case how the fifth parameter is checked changes in the future.

Note: this is a non-blocking comment, but would be very nice to include in this PR.